StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1010CodeFixProvider.cs

```diff
@@ -64,26 +65,25 @@ private static Task<Document> GetTransformedDocumentAsync(Document document, Syn
                 SyntaxToken precedingToken = token.GetPreviousToken();
                 if (precedingToken.TrailingTrivia.Any(SyntaxKind.WhitespaceTrivia))
                 {
-                    SyntaxToken corrected = precedingToken.WithoutTrailingWhitespace().WithoutFormatting();
-                    replacements[precedingToken] = corrected;
+                    replacements.Add(precedingToken);
                 }
             }
 
-            if (!token.TrailingTrivia.Any(SyntaxKind.EndOfLineTrivia) && token.TrailingTrivia.Any(SyntaxKind.WhitespaceTrivia))
+            SyntaxTriviaList trailingTrivia = token.TrailingTrivia;
+            if (!trailingTrivia.Any(SyntaxKind.EndOfLineTrivia) && trailingTrivia.Any(SyntaxKind.WhitespaceTrivia))
             {
-                SyntaxToken corrected = token.WithoutTrailingWhitespace().WithoutFormatting();
-                replacements[token] = corrected;
+                replacements.Add(token);
             }
 
             if (replacements.Count == 0)
             {
-                return Task.FromResult(document);
+                return document;
             }
 
-            var transformed = root.ReplaceTokens(replacements.Keys, (original, maybeRewritten) => replacements[original]);
+            var transformed = root.ReplaceTokens(replacements, (maybeRewritten, _) => maybeRewritten.WithoutTrailingWhitespace().WithoutFormatting());
```

Comment:
:exclamation: The code to the right of `=>` is correct, but the parameter names to the left of `=>` are not correct. The parameters should be `(original, maybeRewritten)`, and `original` (the first parameter) is intentionally not referenced in the expression.